### ISSUE TITLE
check corral quota in case storage allocation request must be made

### ISSUE THREAD

**Issue by jasonfleming** [StormSurgeLive/asgs#939]

**Comment by jasonfleming** [comment]
This is not technically possible and will not be in the foreseeable future.

**Comment by wwlwpd** [comment]
> This is not technically possible and will not be in the foreseeable future.

Correct, per TACC admins. There is no way to check quota or disk usage that meets our needs: `df -h` is too imprecise (location) and `du -h` takes way too long.
